node/tests/RedisClient.test.ts

```diff
@@ -365,6 +365,92 @@ describe("GlideClient", () => {
         TIMEOUT,
     );
 
+    it.each([ProtocolVersion.RESP2, ProtocolVersion.RESP3])(
+        "copy with DB test_%p",
+        async (protocol) => {
+            if (cluster.checkIfServerVersionLessThan("6.2.0")) return;
+
+            const client = await GlideClient.createClient(
+                getClientConfigurationOption(cluster.getAddresses(), protocol),
+            );
+
+            const source = `{key}-${uuidv4()}`;
+            const destination = `{key}-${uuidv4()}`;
+            const value1 = uuidv4();
+            const value2 = uuidv4();
+            const index0 = 0;
+            const index1 = 1;
+            const index2 = 2;
+
+            try {
+                // neither key exists
+                checkSimple(
+                    await client.copyDB(source, destination, index1, false),
+                ).toEqual(false);
+
+                // source exists, destination does not
+                expect(await client.set(source, value1)).toEqual("OK");
+                checkSimple(
+                    await client.copyDB(source, destination, index1, false),
+                ).toEqual(true);
+                expect(await client.select(index1)).toEqual("OK");
+                checkSimple(await client.get(destination)).toEqual(value1);
+
+                // new value for source key
+                expect(await client.select(index0)).toEqual("OK");
+                expect(await client.set(source, value2)).toEqual("OK");
+
+                // no REPLACE, copying to existing key on DB 0 & 1, non-existing key on DB 2
```

Comment:
looks like we are only copying to db 1 here
```suggestion
                // no REPLACE, copying to existing key on DB 1, non-existing key on DB 2
```